### DISCUSSION TITLE
Replacing SSL certificate in hubs clouds

### DISCUSSION THREAD

**Discussion by Frans** [mozilla/hubs#5665] [Hubs Cloud Questions] [answered]
Hi, I have a fully installed hubs clous on aws, via a sub domain that is hosted elsewhere. It appears I might have blown away the SSL certificate for it. How can I replace the SSL certificate for Hubs, without having to reinstall the whole stack?

**Comment by pattersonbl2** [comment, marked as answer]
<img width="927" alt="Screen Shot 2022-08-24 at 3 04 14 PM" src="https://user-images.githubusercontent.com/28871018/186501993-2f532ff3-6191-46c3-85c4-04dc8a97a9e6.png">

You can update your template with a new cert in this location.  this will just restart the stack with the new cert.

**Reply by Frans** [reply]
That looks like the form to deploy the whole stack, is that you are suggesting that I reinstall the stack. 
Or where can I find this form to change the ssl certficate?

**Reply by Frans** [reply]
Ok, found where to update this, for anyone else wondering.
To update the stack, go to cloudformation. 
Open that stack tab, select the root stack for Hubs.
In the top right is a update button, the following pages allows you to update the information that you set during deploy.
